js/src/features/human-resouce/components/HumanResourceCalendar/HumanResourceCalendarDay.tsx

```diff
@@ -15,12 +17,22 @@ export const HumanResourceCalendarDay: React.FC<Props> = ({
   const { className, title } = generateCellInfo(date, absences);
 
   if (!className) {
-    return <td className="timetable-cell" />;
+    return (
+      <td
+        className={`timetable-cell ${
+          isLastDayOfMonth(date) ? "last-day-of-month" : ""
```

Comment:
The same functionality here and below. Calculate it above.

Reply:
Done.